contracts/guard.sol

```diff
@@ -0,0 +1,89 @@
+// guard.sol -- simple whitelist implementation of DSAuthority
```

Comment:
add a resource link at the top if you reference this contact from somewhere else